packages/ui/src/components/JobCard/Progress/Progress.tsx

```diff
@@ -5,35 +5,41 @@ import { Status } from '@bull-board/api/typings/app';
 import { STATUSES } from '@bull-board/api/src/constants/statuses';
 
 interface ProgressProps {
-  percentage: number;
+  progress: number | object;
   strokeWidth?: number;
   status: Status;
   className?: string;
 }
 
-export const Progress = ({ percentage, status, className, strokeWidth = 6 }: ProgressProps) => {
+export const Progress = ({ progress, status, className, strokeWidth = 6 }: ProgressProps) => {
   const commonProps = {
     cx: '50%',
     cy: '50%',
     r: `calc(50% - ${strokeWidth / 2}px)`,
     strokeWidth,
     ['transform-origin']: 'center',
   };
+  
+  const getPercentage = (): number => {
```

Comment:
why do you need it as a function?
Calculate the value, and use it on both places.

Reply:
I have made the requested changes